### PR TITLE
NF-994: Updates changes to reflect newer GoBGP repo

### PR DESCRIPTION
**Pull request by OferMania**
Also replaces a patch with a cleaner build step to "build" proto.go files from the protobuf, as this makes it easier to pull in protobuf changes from osrc source and modify with any of our patches.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ RUN apt-get update \
       automake \
```
Comment (jmunson-stateless) [inline docker/Dockerfile:7]: Should we also roll the stateless/base build to :10 while we are in here?  (Several lines above this.)
Might want to do that as a separate PR, though.

File: patches/series [hunk 2]
```diff
@@ -1,5 +1,4 @@
-# This series applies on GIT commit v2.9.0
+# This series applies on GIT commit 8cadc751ca6cccbf2b8edbab6b11c01b347418eb
```
Comment (jmunson-stateless) [inline patches/series:1]: Is there an associated semver with that SHA, or is this considered to be more precise?

File: patches/series [hunk 3]
```diff
@@ -1,5 +1,4 @@
...
 unix-socket.patch
 set-config-api.patch
 monitor-table-multi.patch
-generated-grpc.patch
```
Comment (jmunson-stateless) [inline patches/series:5]: Do we no longer want generated-grpc.patch?  How do we know that?  If we don't, is there a reason to keep the .patch file around?

File: docker/Dockerfile (hunk 1 above)
Comment (MikeZappa87) [reply docker/Dockerfile:7]: In a effort to not cause a holy war, could we just use the gobgp from osrg?

File: patches/series (hunk 2 above)
Comment (OferMania) [reply patches/series:1]: This is more precise

File: patches/series (hunk 3 above)
Comment (OferMania) [reply patches/series:5]: Apologies for not explaining this earlier: This patch merely updates the "generated" proto.go file from a previous proto patch. But you should never be modifying proto.go files directly. Instead, I undo the patch and add to the build script a step to "build" proto.go out of the proto changes, and do this BEFORE building all the go code. This is arguably correct protocol buffer behavior.
